### PR TITLE
docs: added key types for limit-count plugin

### PR DESCRIPTION
**Pull request by iamayushdas**
### What this PR does / why we need it:
To make users aware of limit-count key types the can use
#3855 

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: docs/en/latest/plugins/limit-count.md [hunk 1]
```diff
@@ -79,7 +79,17 @@ curl -i http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335
 You can open dashboard with a browser: `http://127.0.0.1:9080/apisix/dashboard/`, to complete the above operation through the web interface, first add a route:
 ![Add a router.](../../../assets/images/plugin/limit-count-1.png)
 
-Then add limit-count plugin:
+## Then add limit-count plugin:
```
Comment (spacewander) [inline docs/en/latest/plugins/limit-count.md:82]: Why make it a title?

File: docs/en/latest/plugins/limit-count.md [hunk 2]
```diff
@@ -79,7 +79,17 @@ curl -i http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335
...
+
+The <strong>Key type</strong> in limit-count plugin can be the following:
```
Comment (spacewander) [inline docs/en/latest/plugins/limit-count.md:84]: Please use markdown syntax if possible

**Review by spacewander — CHANGES_REQUESTED** [review]

File: docs/en/latest/plugins/limit-count.md (hunk 1 above)
Comment (iamayushdas) [reply docs/en/latest/plugins/limit-count.md:82]: #### Then add limit-count plugin:

is this size good to go?
to make it lil bit more noticed

File: docs/en/latest/plugins/limit-count.md (hunk 2 above)
Comment (iamayushdas) [reply docs/en/latest/plugins/limit-count.md]: changes done in next commit

**Pull request closed without merging**